app/scenes/Trash/index.tsx

```diff
@@ -2,19 +2,35 @@ import { observer } from "mobx-react";
 import { TrashIcon } from "outline-icons";
 import * as React from "react";
 import { useTranslation } from "react-i18next";
+import Button from "~/components/Button";
 import Empty from "~/components/Empty";
+import Flex from "~/components/Flex";
 import Heading from "~/components/Heading";
 import PaginatedDocumentList from "~/components/PaginatedDocumentList";
 import Scene from "~/components/Scene";
 import Subheading from "~/components/Subheading";
+import { permanentlyDeleteDocumentsInTrash } from "~/actions/definitions/trash";
+import useActionContext from "~/hooks/useActionContext";
 import useStores from "~/hooks/useStores";
 
 function Trash() {
   const { t } = useTranslation();
   const { documents } = useStores();
+  const context = useActionContext();
   return (
     <Scene icon={<TrashIcon />} title={t("Trash")}>
-      <Heading>{t("Trash")}</Heading>
+      <Flex align="baseline" justify="space-between">
+        <Heading>{t("Trash")}</Heading>
+        {documents.deleted.length > 0 && (
```

Comment:
There is an `actions` property on the `<Scene` component to place this button in the correct place, no need for the flex wrapper

Reply:
![Screenshot 2024-04-12 at 8 26 47 AM](https://github.com/outline/outline/assets/132386067/afec6762-1a71-40e6-843f-27f610c8feb9)

The `action` on the `Scene` looks disassociated from the Header at the far-right. Is this alright?
I'd prefer the button to live right next to the Header. What do you think?

Reply:
Yes, it's ok for consistency with other scenes. Might turn into a "..." if other actions get added here.